.evergreen/config.yml

```diff
@@ -326,11 +347,44 @@ axes:
 
 buildvariants:
 
-- matrix_name: "tests-all"
```

Comment:
i couldn't figure out a way to still use the matrix but have it skip the atlas tests on macOS, let me know if you have any ideas

Reply:
see here: https://github.com/mongodb/mongo-python-driver/blob/master/.evergreen/config.yml#L1520

you'll just make a matrix entry that specifies a specific host

Reply:
ah, yeah i was trying to embed these tasks within the existing matrix but adding a new one works too.